socorro/processor/rules/mozilla.py

```diff
@@ -309,12 +359,8 @@ def action(self, raw_crash, dumps, processed_crash, processor_meta):
         if crash_time == submitted_timestamp_epoch:
             processor_notes.append("client_crash_date is unknown")
 
-        # StartupTime: must have started up some time before crash
-        try:
-            startup_time = int(raw_crash.get("StartupTime", crash_time))
-        except ValueError:
-            startup_time = 0
-            processor_notes.append('non-integer value of "StartupTime"')
```

Comment:
We're now doing validation and normalization in the `CopyFromRawCrashRule` so we don't need to validate the data here.